pep_sphinx_extensions/pep_theme/templates/not_found.html

```diff
@@ -0,0 +1,41 @@
+{# 404 Not Found: Redirect typos and link to PEP 0 #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="color-scheme" content="light dark" />
+    <title>PEP Not Found | peps.python.org</title>
+    <link rel="shortcut icon" href="{{ pathto('_static/py.png', resource=True) }}"/>
+    <link rel="stylesheet" href="{{ pathto('_static/style.css', resource=True) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/mq.css', resource=True) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/dark.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: dark)" id="css-dark"/>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+    <meta name="description" content="Python Enhancement Proposals (PEPs)"/>
+    <script>
+    const prefixNum = window.location.pathname.slice(1).replace(/^(\d*)$/, "$1")
+    if (!isNaN(+prefixNum)) window.location.pathname = "/pep-" + prefixNum.padStart(4, "0")
```

Comment:
```suggestion
    const match = window.location.pathname.match(/pep-(\d+)/)
    if (match != null) {
        const pepNum = match[1]
        window.location.pathname = "/pep-" + pepNum.padStart(4, "0")
    }
```
This should achieve the same thing and it's IMHO a bit clearer and flexible.

Reply:
I don't want to include the `pep-` prefix for the auto-redirects, partly for laziness as it means I can now just type `/X` instead of `/pep-000X` etc.

Happy to use your approach although I'll tweak a little.

A